### PR TITLE
Remove cookie based binding from webapps in console

### PR DESCRIPTION
**Pull request by Rashmini**
### Purpose
$subject

### Related Issues
- None

### Checklist
- [x] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- None

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 1]
```diff
@@ -259,6 +262,16 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
         }
     }, [ template ]);
 
+    useEffect(() => {
```
Comment (brionmario) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:265]: Add a doc comment explaining this hook.

**Review by brionmario — APPROVED** [review]

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 2]
```diff
@@ -477,7 +490,7 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                         label: "Opaque",
                         value: ele
                     });
-                } else if ((isSPAApplication) && isBinding && ele === "cookie") {
+                } else if ((isSPAApplication || isOIDCWebApplication) && isBinding && ele === "cookie") {
```
Comment (brionmario) [inline apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:493]:
```suggestion
                // Cookie binding was hidden from the UI for SPAs & Traditional OIDC with https://github.com/wso2/identity-apps/pull/2254/files
                } else if ((isSPAApplication || isOIDCWebApplication) && isBinding && ele === "cookie") {
```

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx (hunk 1 above)
Comment (Rashmini) [reply apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:271]: fixed with b02cac6a564ad16853cc79e541e354bc911f9a7b

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx (hunk 2 above)
Comment (Rashmini) [reply apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:500]: fixed with b02cac6a564ad16853cc79e541e354bc911f9a7b

**Review by DinikaSen — APPROVED** [review]

**Pull request merged**